blockly-rtc/server/websocket_server.js

```diff
@@ -59,4 +59,20 @@ async function onConnect(user) {
     const entries = await database.query(serverId);
     callback(entries);
   });
+
```

Comment:
You should probably document these functions.

Reply:
Yes, this would be helpful to the user. I have now created handler functions and added JSDocs.